### ISSUE TITLE
On my own dataset, the issue of map being 0

### ISSUE THREAD

**Issue by 1781686573** [LSH9832/edgeyolo#35]
20230626_220743 edgeyolo.train.trainer:379 - Start Train Epoch 1 
20230626_220826 edgeyolo.train.trainer:457 - epoch:1/150 iter:100/488 mem:5829MB t_iter:1.04 lr:3.149e-06 loss:{total:9.27 iou:3.17 conf:5.53 cls:0.57} ETA:8:36:33
20230626_220908 edgeyolo.train.trainer:457 - epoch:1/150 iter:200/488 mem:5829MB t_iter:0.96 lr:1.260e-05 loss:{total:7.25 iou:3.01 conf:3.52 cls:0.72} ETA:8:37:18
20230626_220950 edgeyolo.train.trainer:457 - epoch:1/150 iter:300/488 mem:5831MB t_iter:1.17 lr:2.834e-05 loss:{total:6.50 iou:2.50 conf:3.47 cls:0.53} ETA:8:33:24
20230626_221032 edgeyolo.train.trainer:457 - epoch:1/150 iter:400/488 mem:5831MB t_iter:0.88 lr:5.039e-05 loss:{total:4.61 iou:1.99 conf:1.98 cls:0.65} ETA:8:30:49
20230626_221108 edgeyolo.models:146 - weight file saved to output/train/tiny_lrelu/last.pth
20230626_221108 edgeyolo.models:146 - weight file saved to output/train/tiny_lrelu/epoch_001.pth
0it [00:07, ?it/s]
20230626_221116 edgeyolo.train.val.coco_evaluator:208 - Evaluate in main process...
20230626_221116 edgeyolo.train.trainer:523 - 
Average forward time: 0.00 ms, Average NMS time: 0.00 ms, Average inference time: 0.00 ms

20230626_221116 edgeyolo.models:146 - weight file saved to output/train/tiny_lrelu/last.pth
20230626_221116 edgeyolo.models:146 - weight file saved to output/train/tiny_lrelu/epoch_001.pth
Why is the NMS time and inference time showing zero on my own dataset?And，map0.5 and map0.5:0.95 of evaluation on my model are zero.

**Comment by LSH9832** [comment]
it means it detects nothing in validate dataset.
here are some suggestions.
1. if there's only small amount of images in your dataset, use the pretrained model in coco dataset at start, or you can hardly get a good result.
2. continue training and wait for another several epochs and then see whether it is still zero.
